ibis/expr/operations.py

```diff
@@ -740,6 +741,9 @@ class MultiQuantile(Quantile):
                       name='interpolation',
                       default='linear')]
 
+    def output_type(self):
```

Comment:
I think most of these are operation specific so I don't know if there's any consolidation possible here.